client/ayon_resolve/api/utils.py

```diff
@@ -40,7 +40,8 @@ def get_resolve_module():
                 )
             )
         elif sys.platform.startswith("linux"):
-            expected_path = "/opt/resolve/libs/Fusion/Modules"
+            resolve_root = os.getenv("RESOLVE_ROOT", "/opt/resolve")
+            expected_path = f"{resolve_root}/libs/Fusion/Modules"
```

Comment:
I think this would need to apply to all platforms - not just linux. Right?
Or is there a reason why this _may_ only apply to Linux?